app/views/products/show.html.erb

```diff
@@ -1,5 +1,12 @@
 <h1><%= @product.name %></h1>
-<h3>$<%= @product.price %></h3>
+<h3>List Price: US $<%= @product.price %></h3>
+<% if @product.quantity <= 5 %>
+  <li><span class="quant">Limited - Only <%= @product.quantity %> left in stock </span></li>
+  <li><span class="small">Ships from and sold by: <%= @product.name %> </span></li>
+<% else %>
+  <span class="small">Ships from and sold by: <%= @product.name %> </span>
```

Comment:
Can move the code below outside the if statement since it should always appear:
`<span class="small">Ships from and sold by: <%= @product.name %> </span>`